modules/serverless-logic-install-kn-workflow-artifact-images.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// * /serverless/install/serverless-logic-install-kn-workflow-plugin-cli.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-logic-install-kn-workflow-artifact-images_{context}"]
+= Installing the OpenShift Serverless Logic Knative Workflow plugin using the artifacts image
+
+.Prerequisites
+
+* You have installed the Knative (`kn`) CLI.
+* You have installed Podman on your local machine. 
+
+.Procedure
+
+. Download the Knative Workflow plugin using the `logic-kn-workflow-cli-artifacts-rhel8` image by running the following commands:
+
+.. To login to the Red{nbsp}Hat Registry run the following command on a system with podman installed:
++
+[source,shell]
+----
+$ podman login registry.redhat.io
+----
++
+You can use your Red{nbsp}Hat customer portal or a registry service account username and password.
+
+.. Start the `logic-kn-workflow-cli-artifacts-rhel8` image.
++
+[source,shell]
+----
+$ export KN_IMAGE=registry.redhat.io/openshift-serverless-1/logic-kn-workflow-cli-artifacts-rhel8:1.33.0
+----
++
+[source,shell]
+----
+$ export KN_CONTAINER_ID=$(podman run -di $KN_IMAGE)
+----
```

Comment:
this appears to be two separate steps for setting two variables.

```suggestion
.. Set a variable for [whatever this is] by running the following command:
+
[source,shell]
----
$ export KN_IMAGE=registry.redhat.io/openshift-serverless-1/logic-kn-workflow-cli-artifacts-rhel8:1.33.0
----

.. Set a variable for [whatever this is] by running the following command:
+
[source,shell]
----
$ export KN_CONTAINER_ID=$(podman run -di $KN_IMAGE)
----
```

Reply:
Alright! I have added this